core/dictgen/src/rootcling_impl.cxx

```diff
@@ -4349,6 +4349,8 @@ int RootClingMain(int argc,
 
    // For the list of 'opaque' typedef to also include string, we have to include it now.
    interp.declare("#include <string>");
+   // For initializing TNormalizedCtxt.
+   interp.declare("#include <RtypesCore.h>");
```

Comment:
How did this work in the past for genreflex (which IIUC doesn't include `Rtypes.h`)?

Reply:
It works/worked because we include Rtypes.h somewhere above. This like just makes the init requirement of TNormalizedContext more explicit.